word-search-page/word-search-page.css

```diff
@@ -0,0 +1,164 @@
+/* THis is general styling for mobile window */
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    outline: none;
+}
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #DFD1E0;
+}
+.whole-page {
+    width: 100%;
+    height: 97%;
+}
+/* This is for the page content to be in the middle */
+.main-container {
+    padding: 25px;
+    width: 100%;
+    height: 100vh;
+    margin: auto;
+}
+/* This is title styling */
+.header {
+    margin-top: 40px;
+    padding: 8px;
+    border-radius: 20px;
+    text-align: center;
+    background-color: #905983;
+    color: #fff;
+    width: 95%;
+    height: 70px;
+    line-height: 60px;
+    margin-right: 10px;
+    font-size: 2.5rem;
+}
+/* This is image styling */
+.image {
+    width: 85%;
+    height: 30%;
+    margin-right: 10px;
+}
+.photo{
+    margin-top: 15px;
+    width: 109%;
+    height: 95%;
+    border-radius: 45px;
+}
+/* This is search box styling */
+.search-container {
+    padding: 15px;
+    width: 100%;
+    height: 70%;
+    margin-top: 20px;
+}
+.word-search {
+    text-align: right;
+    color: #905983;
+    font-size: 1.6rem;
+    font-weight: 800;
+    margin: 5px 10px;
+}
+.search-input {
+    color: white;
+    background-color: #905983;
+    width: 54%;
+    height: 3rem;
+    border: none;
+    outline: none;
+    font-size: 2rem;
+    border-radius: 15px;
+    text-align: right;
+    padding: 10px;
+    margin: 5px 10px 5px 5px;
+}
+.search-button {
+    color: white;
+    background-color: #905983;
+    font-size: 1.5rem;
+    line-height: 2rem;
+    width: 30%;
+    height: 3rem;
+    border: none;
+    outline: none;
+    border-radius: 15px;
+    padding: 10px;
+    margin: 5px;
+    text-align: center;
+    font-weight: 600;
+}
+::placeholder { 
+    color: white;
+  }
+.search-result {
+    list-style: none;
+    background-color: #905983;
+    color: white;
+    border-radius: 15px;
+    padding: 10px;
+    width: 93%;
+    height: 50%;
+    margin: 15px 10px;
+    font-size: 2rem;
+    font-weight: 600;
+}
+
+/* responsive styling for laptop window*/
+
```

Comment:
** General point of laptop screen style **

- Fix the style of the laptop screen , you have a horizontal scroll you have to fix it.